doc/lua_api.md

```diff
@@ -7776,7 +7778,10 @@ Global tables
       Note: changes to initial properties will only affect entities spawned afterwards,
       as they are only read when spawning.
 * `core.object_refs`
-    * Map of object references, indexed by active object id
+    * Map of active object references, indexed by active object id
+    * Deprecated: Use `core.objects_by_guid` instead.
```

Comment:
"Deprecated" is weird here, as the engine will still keep it for its own use-cases.
How about turning this into a recommendation?

Reply:
:shrug:

that's what i would have done initially, but i don't see much of a point. it was a mistake to expose active object ids (as keys of this table) to begin with.

so from a modder perspective, `core.objects_by_guid` is strictly better and this table should not be used, because active object ids should not be used.

whether the engine keeps this table, or whether we remove it, doesn't matter; there are plenty of engine APIs we do not want modders to use. and deprecated communicates that. (there doesn't have to be a deprecation warning, though: there's little point in removing this.)

but i'm also fine with wording like "strongly recommended" or "obsolete" :shrug:

Reply:
called it obsolete